app/src/main/java/com/iic/shopingo/services/CurrentUser.java

```diff
@@ -9,10 +9,10 @@
 public class CurrentUser {
 
   public enum State {
-    LOGGED_OUT,
-    LOGGED_IN,
-    SHOPPING,
-    REQUESTING
+    IDLE,
+    TRIPPING,
+    REQUESTING,
+    LOGGED_OUT
```

Comment:
`LOGGED_OUT` should probably be 0, since it's the default state (until we verify that you're logged in).